lib/ns.c

```diff
@@ -172,6 +200,20 @@ static struct ns *ns_lookup(ns_id_t ns_id)
 	return (RB_FIND(ns_head, &ns_tree, &ns));
 }
 
+/* Look up a NS by name */
+static struct ns *ns_lookup_name(const char *name)
+{
+	struct ns *ns = NULL;
+
+	RB_FOREACH(ns, ns_head, &ns_tree) {
```

Comment:
`RB_FOREACH (...`

Lot's of places this needs to be fixed

Reply:
I don't get it.
the checkstyle script does not permit a space between RB_FOREACH and '('.
So what is the problem please ?

Reply:
https://www.kernel.org/doc/html/v4.10/process/coding-style.html

Section 3.1 -> Spaces after a `for (`.  Yes our code is inconsistent but I want to make it consistent.  Since these are wrappers for for loops then I would like a space.